lib/rubocop/cop/util.rb

```diff
@@ -286,7 +286,7 @@ def line_distance(n1, n2)
         n2.loc.line - n1.loc.line
       end
 
-      def preceed?(n1, n2)
+      def precede?(n1, n2)
```

Comment:
Watch out for this one

Reply:
Probably it is safe, a method call for `preceed` is rewritten ( https://github.com/bbatsov/rubocop/pull/5144/files#diff-003ef0dbafbc9a7c26a720820f5b200bR27 ), and I can't find `preceed` from our codebase with `git grep`